tests/acceptance/features/bootstrap/TagContext.php

```diff
@@ -104,6 +103,26 @@ public function theUserHasCreatedFollowingTags(string $user, string $fileOrFolde
 		$this->featureContext->theHttpStatusCodeShouldBe(200);
 	}
 
+	/**
+	 * @Given /^user "([^"]*)" has tagged the following (folders|files) of the space "([^"]*)":$/
+	 *
+	 * @param string $user
+	 * @param string $filesOrFolders (files|folders)
+	 * @param string $space
+	 * @param TableNode $table
+	 *
+	 * @return void
+	 * @throws Exception
+	 */
+	public function userHasCreatedTheFollowingTagsForFilesOfTheSpace(string $user, string $filesOrFolders, string $space, TableNode $table):void {
+		$rows = $table->getHash();
```

Comment:
check that `path` and `tagName` columns are in the data table
```suggestion
	public function userHasCreatedTheFollowingTagsForFilesOfTheSpace(string $user, string $filesOrFolders, string $space, TableNode $table):void {
		$this->featureContext->verifyTableNodeColumns($table, ["path", "tagName")
		$rows = $table->getHash();
```